redis.conf

```diff
@@ -186,12 +186,13 @@ logfile ""
 databases 16
 
 # By default Redis shows an ASCII art logo only when started to log to the
-# standard output and if the standard output is a TTY. Basically this means
-# that normally a logo is displayed only in interactive sessions.
+# standard output and if the standard output is a TTY and syslog logging is
+# disabled. Basically this means that normally a logo is displayed only in
+# interactive sessions.
 #
 # However it is possible to force the pre-4.0 behavior and always show a
 # ASCII art logo in startup logs by setting the following option to yes.
-always-show-logo yes
+always-show-logo no
```

Comment:
```suggestion
# always-show-logo yes
```

note that this change is not just fixing a comment, it actually changes the behavior when someone uses the default conf file.

on the other hand, i agree that this needs to change since most of the not commented lines in this file represent the default value.
other times they have a non default value but are put in a comment.

my suggestion leads to the same result, but maybe you agree that it's nicer?

Reply:
Maybe we could change the default value instead?